x/ibc-hooks/types/msgs.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// constants.
+const (
+	TypeMsgEmitIBCAck = "emit-ibc-ack"
+)
+
+var _ sdk.Msg = &MsgEmitIBCAck{}
+
+func (m MsgEmitIBCAck) Route() string { return RouterKey }
+func (m MsgEmitIBCAck) Type() string  { return TypeMsgEmitIBCAck }
+func (m MsgEmitIBCAck) ValidateBasic() error {
+	_, err := sdk.AccAddressFromBech32(m.Sender)
+	if err != nil {
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "Invalid sender address (%s)", err)
+	}
+	return nil
+}
```

Comment:
Does it make sense to do additional validation of packet sequence and channel here?

Reply:
I don't think we do. The packet sequence is an uint64, and the channel can be any string (even if they generally are `channel-<number>`